### PR TITLE
Feature/feature register user

### PR DESCRIPTION
**Pull request by xavier-campos-diaz**

### CODE REVIEW

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 7
           

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/activities/MainActivity.java  1
- app/src/main/java/org/pesmypetcare/mypetcare/activities/LoginActivity.java  2
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/SignUpFragment.java  5
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/LogInFragment.java  1
- app/src/main/java/org/pesmypetcare/mypetcare/User.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5212442&bid=17092124)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 7
           

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/LogInFragment.java  1
- app/src/main/java/org/pesmypetcare/mypetcare/User.java  1
- app/src/main/java/org/pesmypetcare/mypetcare/activities/MainActivity.java  1
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/SignUpFragment.java  5
- app/src/main/java/org/pesmypetcare/mypetcare/activities/LoginActivity.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5212442&bid=17092124)

**Review by santidrj — CHANGES_REQUESTED** [review]
@xavier-campos  the User class is not documented

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/LogInFragment.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                loginUser();
+            } else {
+                testToast("Incorrect entry");
+            }
+
+        });
+        return view;
+    }
+
+    private void testToast(String s) {
```
Comment (AlbertPG) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/LogInFragment.java:51]: You need to comment this function as well

**Review by AlbertPG — CHANGES_REQUESTED** [review]
After fixing this issue, you have got my ok

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/User.java  1
- app/src/main/java/org/pesmypetcare/mypetcare/activities/LoginActivity.java  2
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/SignUpFragment.java  5
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/LogInFragment.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5212442&bid=17092124)

**Pull request merged**